### PR TITLE
Update draft-tigress-requirements.md

### PR DESCRIPTION
**Pull request by castiz**

### CODE REVIEW

**Comment by castiz** [comment]
This is in regards to this issue: https://github.com/dimmyvi/tigress-requirements/issues/25. I'm moving the signal pieces out of its own section and into the section for messaging application review.

File: draft-tigress-requirements.md [hunk 1]
```diff
@@ -186,18 +186,13 @@ A number of existing solutions / protocols have been reviewed in order to be use
 
 ## Arbitrary Messaging Channel (Email / WhatsApp / SMS / Signal / etc.)
 
-The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel that supports binary file transfer, but this would not support provisioning flows which require multiple round trips as requied by (Req-RoundTrips).
+The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel that supports binary file transfer, but this would not support provisioning flows which require multiple round trips as requied by (Req-RoundTrips). The same requirement applies to Signal protocol outside of the Signal app, as the Req-RoundTrips would likely be difficult and add a lot of friction for the user.
```
Comment (dimmyvi) [inline draft-tigress-requirements.md:189]: I think reference Signal is not appropriate here, the requirement here applies to messaging application, the paragraph about Signal - to a messaging protocol. Signal protocol may be (and probably have been) implemented in applications other than Signal Messaging Application

Comment (dimmyvi) [reply draft-tigress-requirements.md]: What could be added here though, is a reference to the requirement of separation of channels - but we decided to remove this requirement...

File: draft-tigress-requirements.md [hunk 2]
```diff
@@ -186,18 +186,13 @@ A number of existing solutions / protocols have been reviewed in order to be use
...
 
 ## GSS-API, Kerberos
 
 GSS-API {{!RFC2078}} and Kerberos {{!RFC4120}} are authentication technologies which could be used to authenticate Sender, Receiver and intermediary. However, as they provide strong authentication, they would allow the Intermediary server to build a social graph in violation of (Req-Privacy). Their setup also require strong coordination between the actors of the system which seems overly costly for the intended system.
 AWS S3 could be used as an Intermediary server but it would force all participants to use a specific cloud service which is in violation of (Req-AnyPlatorm).
 
-## Signal Protocol
```
Comment (alexpell00) [inline draft-tigress-requirements.md:196]: I think we could merge this section into the "## GSS-API, Kerberos" section, or have a new section for "Arbitrary Secure Messaging Protocol".

**Pull request merged**